joplin/importer/page_importer.py

```diff
@@ -33,7 +38,7 @@ def fetch_page_data(self):
             # headers={
             #     "Content-type": "application/json",
             # },
-            verify=False
+            verify=True
```

Comment:
just curious as to what prompted setting verify here, pretty sure it being set to false was pulled from some example somewhere

Reply:
It was throwing weird messages during pytest without it. And all of our api endpoints are https, so we're good